pkg/adapter/awssqs/adapter.go

```diff
@@ -63,9 +60,17 @@ func (a *Adapter) Start(ctx context.Context, stopCh <-chan struct{}) error {
 
 	logger.Info("Starting with config: ", zap.Any("adapter", a))
 
+	urlParts := strings.Split(a.QueueURL, ".")
```

Comment:
I would move this into a function `getRegion(a.QueueURL)` and provide an example URL to show what is expected (e.g. `https://sqs.us-east-2.amazonaws.com/123456789012/MyQueue`).

Reply:
Good point, I should have reviewed this after that change, thanks.

Reply:
https://github.com/knative/eventing-sources/pull/124/commits/573ca8b6814d24cd42ef3c01f50a7ec7bd76106b